test/karma.debug.conf.js

```diff
@@ -141,4 +141,4 @@ module.exports = function(config) {
     var bs = grunt.option('browsers').split(/,/).map(function(b) { return b.trim(); });
     config.browsers = bs;
   }
-};
\ No newline at end of file
+};
```

Comment:
This file is just whitespace changes. Please don't commit this with just whitespace changes.